<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anatel_dialog</class>
 <widget class="QDialog" name="anatel_dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base de Dados Anatel</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="main_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="anatel_table">
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>37</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row/>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STATUS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ENTITY</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>FISPEL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SERVICE NUMBER</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RF ACT</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>STATION NUMBER</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ADDRESS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>UF</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>COUNTY</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ISSUE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TECHNOLOGY</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>HALF ACCESS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>START FREQUENCY (MHz)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>END FREQUENCY (MHz)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>AZIMUTH</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SEASON TYPE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CLASS. INFRA PHYSICAL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>INFRA PHYSICAL SHARING</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>INFRA PHYSICAL SHARING DISP</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ANTENNA TYPE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PREVIOUS APPROVAL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ANTENNA GAIN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>FRONT COAST</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>1/2 POINT ANGLE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ELEVATION</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>POLARIZATION</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ANTENNA HEIGHT (m)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TRANSMISSION APPROVAL (W)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TRANSMISSION POWER</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LATITUDE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LONGITUDE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LATITUDE DMS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>LONGITUDE DMS</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CODE DEBIT TFI</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>PRIMARY DATE LICENSED</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CREATED IN</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="13">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="14">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="15">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="16">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="20">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="21">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="22">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="23">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="24">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="25">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="26">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="27">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="28">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="30">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="31">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="35">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="bottom_frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLabel" name="label_last_update">
           <property name="text">
            <string>Anatel Database</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="update_database_button">
           <property name="text">
            <string>Update Database</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="combo_box_state">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Select State</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QProgressBar" name="progress_bar_anatel">
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%p% completed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="combo_box_county">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Select City</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
